scripts/development/m3_stack/m3coordinator.yml

```diff
@@ -55,3 +55,7 @@ carbon:
 
```

Comment:
Either add this to all configs or remove since it defaults

Reply:
I'd err towards default